jenkins/lava-boot-v2.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*
+#
+
+import os
+from subprocess import Popen
+
+labs = {
+    'lab-free-electrons': {
+        'blacklist': [
+            'android'
+        ],
+    },
+}
+
+# Basic env
+LAB = os.environ['LAB']
+API = os.environ['API']
+API_TOKEN = os.environ['API_TOKEN']
+STORAGE = os.environ['STORAGE']
+CALLBACK = os.environ['CALLBACK']
+GIT_DESCRIBE = os.environ['GIT_DESCRIBE']
+TREE = os.environ['TREE']
+BRANCH = os.environ['BRANCH']
+ARCH = os.environ['ARCH']
+
+# Tokens
+LAVA_TBAKER_TOKEN = os.environ['LAVA_TBAKER_TOKEN']
+LAVA_MHART_TOKEN = os.environ['LAVA_MHART_TOKEN']
+LAVA_COLLABORA_TOKEN = os.environ['LAVA_COLLABORA_TOKEN']
+LAVA_BAYLIBRE_TOKEN = os.environ['LAVA_BAYLIBRE_TOKEN']
+LAVA_BAYLIBRE_SEATTLE_TOKEN = os.environ['LAVA_BAYLIBRE_SEATTLE_TOKEN']
+LAVA_FREE_ELECTRONS_TOKEN = os.environ['LAVA_FREE_ELECTRONS_TOKEN']
+
+# Command making the jobs
+make_jobs_args = [
```

Comment:
I think it would make more sense to import the lava-v2-jobs-from-api module (might need to be renames...) and call its main function directly rather than calling Python in a shell from Python.

In fact, maybe this can all be stored in the config file used in lava-v2-jobs-from-api and the script could be adjusted to directly read the information, rather than adding an extra level on top of it.  There could be a section for each lab in the config file or something like this.